cmd/check-spelling/data/acronyms.txt

```diff
@@ -17,6 +18,8 @@ CPUID/AB
 CRI/AB
 CVE/AB
 DAX/AB
+DinD/B # Docker in Docker
+dind/B
```

Comment:
Do we need both of these? The lower-case version reminds me of the French *dinde* (which is amusing given the season :turkey: :smile:)